configs/sites/tier1/nas/packages_oneapi.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
+  all:
+    compiler:: [oneapi@2024.2.0,gcc@12.3.0]
+    providers:
+      mpi:: [mpt@2.30]
+  mpi:
+    buildable: False
+  mpt:
+    externals:
+    - spec: mpt@2.30%oneapi@=2024.2.0
+      prefix: /nasa/hpe/mpt/2.30_rhel810
+  intel-oneapi-mkl:
+    externals:
+    - spec: intel-oneapi-mkl@2024.2.0%oneapi@2024.2.0
+      prefix: /nobackup/gmao_SIteam/intel/oneapi
+  wget:
+    buildable: False
+    externals:
+    - spec: wget@1.19.5
+      prefix: /usr
```

Comment:
This is a duplicate entry, it's already in your packages.yaml

Reply:
Ah. Yep. Issues with copy-paste